.github/workflows/pull_request.yml

```
@@ -0,0 +1,30 @@
+name: Pull Request
+concurrency:
+  group: pr_${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

Use Node 16 by default

```suggestion
    - uses: actions/checkout@v3
```